aws/aws-cloudwatch-alarm-expression/main.tf

```
@@ -0,0 +1,46 @@
+resource "aws_cloudwatch_metric_alarm" "alarm" {
+  alarm_name = var.alarm_name
+
+  # We need to 'smuggle' our name_prefix for the package back to massdriver
+  # so we can show the alarms on the correct manifest in the UI
+  alarm_description = jsonencode({
+    name_prefix = var.md_metadata.name_prefix
+    message     = var.message
+  })
```

```suggestion
  alarm_description = var.message
```

While we're in here, I think we can change this to `alarm_description = var.message`. The backend used to use `name_prefix` here but I'm pretty sure it hasn't for a while.

[In the backend](https://github.com/massdriver-cloud/massdriver/blob/06577fab1fb81c13b7fdfcd6bf27e8ed725321e2/lib/massdriver/auto_sre/auto_sre.ex#L226), it's parsed to JSON and if that's successful, it ignores the `name_prefix`. If the parse isn't successful, it treats the entire description as a string.

@xpositivityx can chime in.

---

Yeah, this was a hack to smuggle package names back to MD for a DB lookup. We have since solved that by storing alarms with a resource Id from the cloud. It should just be a string.

---

Good find! Change commited.